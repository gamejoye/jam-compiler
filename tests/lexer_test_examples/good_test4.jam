Class Main {
  let x: Int = 10;
  main(): Bool {
    let status: Bool = x >= 100;
    return status;
  };
};