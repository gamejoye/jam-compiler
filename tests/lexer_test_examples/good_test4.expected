CLASS
TYPEID
{
LET
OBJECTID
:
TYPEID
=
INT_CONST
;
OBJECTID
(
)
:
TYPEID
{
LET
OBJECTID
:
TYPEID
=
OBJECTID
RE
INT_CONST
;
RETURN
OBJECTID
;
}
;
}
;